astropy/io/ascii/latex.py

```diff
@@ -80,13 +82,21 @@ class LatexSplitter(core.BaseSplitter):
     '''
     delimiter = '&'
 
+    def __call__(self, lines):
+        last_line = RE_COMMENT.split(lines[-1])[0].strip()
+        if not last_line.endswith(r'\\'):
+            print(last_line)
+            lines[-1] = last_line + r'\\'
+
+        return super(LatexSplitter, self).__call__(lines)
+
     def process_line(self, line):
         """Remove whitespace at the beginning or end of line. Also remove
         \\ at end of line"""
-        line = line.split('%')[0]
+        line = RE_COMMENT.split(line)[0]
         line = line.strip()
-        if line[-2:] == r'\\':
-            line = line.strip(r'\\')
+        if line.endswith(r'\\'):
+            line = line.rstrip(r'\\')
```

Comment:
@MSeifert04 - yup, that's correct.  But in the interest not clogging up CI even more I'm merging this as-is.  After the 1.3 frenzy quiets we can do this clean up.